data-propagation.md

```diff
@@ -23,12 +26,20 @@ type BlockMsg struct {
 
 The array of message cids must match the `Messages` field in the block when used to construct a [sharray](sharray.md).
 
-Each Filecoin node sets a 'validation function' for the blocks topic that checks that the block is properly constructed, its ticket is valid, the block signature is valid, the miner is a valid miner, and the block is a child of a known good tipset. (TODO: clarify which of these checks are needed, any slowness here impacts propagation time significantly, this is not a full validity check) If an invalid block is received, the peer it was received from should be marked as potentially bad (TODO: we could blacklist peers who send bad blocks, maybe need support from libp2p for this?)
+Every `BlockMsg` received must be validated before being propagated again. If validation fails, it must not be propagated. The validation must check that is its ticket is valid, the block signature is valid, the miner is a valid miner, and the block is a child of a known good tipset.
+
+
+{{% notice todo %}} 
+clarify which of these checks are needed, any slowness here impacts propagation time significantly, this is not a full validity check.
+{{% /notice %}}
```

Comment:
Perhaps reference `validation.md`, where I think I've answered these questions in https://github.com/filecoin-project/specs/pull/289/files#diff-eff160cdcda8f026fba936da7f3fea15

Reply:
I removed it here and put it into the validation.md for this PR. we can link to that once both PRs converge.